client/extensions/woocommerce/state/data-layer/ui/payments/index.js

```diff
@@ -4,124 +4,14 @@
  * External dependencies
  */
 
-import { translate } from 'i18n-calypso';
-import { isEmpty, isEqual } from 'lodash';
+import { isEmpty } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import { WOOCOMMERCE_PAYMENT_ACTION_LIST_CREATE } from 'woocommerce/state/action-types';
-import {
```

Comment:
Curious why are all these changes in this PR?

Reply:
A little bit `prettier` but mostly to split the `getSaveSettingsActionListSteps` reducer out as a separate importable item